core/cli/bin/run

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env node
 
+/* eslint-disable @typescript-eslint/no-var-requires --
+ * this is raw CJS so the lint doesn't apply
+ */
```

Comment:
**thought**: we're not even linting it anyway because it doesn't have a file extension 😅

Reply:
ah, well my text editor is linting it! (it inferred the filetype from the shebang)